third_party/terraform/tests/resource_storage_bucket_test.go

```diff
@@ -829,10 +829,8 @@ func TestAccStorageBucket_website(t *testing.T) {
 	t.Parallel()
 
 	bucketSuffix := acctest.RandomWithPrefix("tf-website-test")
-
-	websiteKeys := []string{"website.0.main_page_suffix", "website.0.not_found_page"}
-	errMsg := fmt.Sprintf("one of `%s` must be specified", strings.Join(websiteKeys, ","))
-	fullErr := fmt.Sprintf("config is invalid: 2 problems:\n\n- \"%s\": %s\n- \"%s\": %s", websiteKeys[0], errMsg, websiteKeys[1], errMsg)
+	errMsg := fmt.Sprintf("one of `(?:%[1]s,%[2]s)|(?:%[2]s,%[1]s)`must be specified",
```

Comment:
want to just hardcode `website.0.main_page_suffix` and `website.0.not_found_page`? I had to look up what `%[1]s` meant and we're not saving much effort interpolating them